console/core/src/main/java/org/eclipse/kapua/app/console/core/client/NorthView.java

```diff
@@ -197,16 +197,21 @@ public void onSuccess(Void arg0) {
                                     gwtSettingService.getSsoLogoutUri(currentSession.getSsoIdToken(),
                                             new AsyncCallback<String>() {
 
-                                        @Override
-                                        public void onFailure(Throwable caught) {
-                                            FailureHandler.handle(caught);
-                                        }
-
-                                        @Override
-                                        public void onSuccess(String result) {
-                                            Window.Location.assign(result);
-                                        }
-                                    });
+                                                @Override
```

Comment:
Can you please change the header to "2020"? I hope I am not too late 😄

Reply:
done!